arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -0,0 +1,1202 @@
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/dnszones/{zoneName}/{recordType}/{relativeRecordSetName}": {
+      "patch": {
+        "tags": [
+          "RecordSets"
+        ],
+        "operationId": "RecordSets_Update",
+        "description": "Creates a RecordSet within a DNS zone.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "zoneName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the zone without a terminating dot."
+          },
+          {
+            "name": "recordType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of DNS record.",
+            "enum": [
+              "A",
+              "AAAA",
+              "CNAME",
+              "MX",
+              "NS",
+              "PTR",
+              "SOA",
+              "SRV",
+              "TXT"
+            ],
+            "x-ms-enum": "RecordType"
+          },
+          {
+            "name": "relativeRecordSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the RecordSet, relative to the name of the zone.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RecordSetUpdateParameters"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate operation."
+          },
+          {
+            "name": "IfMatch",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "The etag of Zone."
+          },
+          {
+            "name": "IfNoneMatch",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Defines the If-None-Match condition. Set to '*' to force Create-If-Not-Exist. Other values will be ignored."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/RecordSet"
+            }
+          }
+        }
```

Comment:
Please specify the "default" response in every operation. Please the docs over [here] (https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#Responses) to understand the importance of default and how negative responses are processed.